source/MaterialXTest/MaterialXRenderGlsl/RenderGlsl.cpp

```diff
@@ -630,21 +630,23 @@ bool GlslShaderRenderTester::runRenderer(const std::string& shaderName,
     return true;
 }
 
-void GlslShaderRenderTester::runBake(mx::DocumentPtr doc, const mx::FileSearchPath& imageSearchPath, const mx::FilePath& outputFileName,
-                                      unsigned int bakeWidth, unsigned int bakeHeight, bool bakeHdr, std::ostream& log)
+void GlslShaderRenderTester::runBake(mx::DocumentPtr doc, const mx::FileSearchPath& imageSearchPath, const mx::FileSearchPath& codeSearchPath, 
+                                     const mx::FilePath& outputFileName, const GenShaderUtil::TestSuiteOptions::BakeSetting& bakeOptions,
+                                     std::ostream& log)
 {
-    mx::ImageVec imageVec = _renderer->getImageHandler()->getReferencedImages(doc);
-    auto maxImageSize = mx::getMaxDimensions(imageVec);
-    bakeWidth = std::max(bakeWidth, maxImageSize.first);
-    bakeHeight = std::max(bakeHeight, maxImageSize.second);
+    const unsigned int bakeWidth = std::max(bakeOptions.resolution, (unsigned int) 2);
```

Comment:
Is this correct. Shouldn't this compare max bakeoptions vs max dimensions on the set of images instead of 2 ?

Reply:
Yes, corrected it.